core/src/main/scala/org/apache/spark/sql/CachedDataFrame.scala

```diff
@@ -409,18 +409,22 @@ object CachedDataFrame
   override def apply(context: TaskContext,
       iter: Iterator[InternalRow]): PartitionResult = {
     var count = 0
-    val buffer = new Array[Byte](4 << 10) // 4K
+    val buffer = new Array[Byte](4 << 10)
+    // 4K
```

Comment:
Idea autoformat messes this. Perhaps better to remove that dangling comment since it is obvious.